spec/react.test.dom.js

```diff
@@ -32,24 +32,30 @@ describe('App.jsx component', () => {
     expect(wrapper.find('MovieCarousel').html()).toBe(null);
   })
 
-  test('Clicking a Person component should call fetch()', async (done) => {
-    await wrapper.find('div.headshot').first().simulate('click');
-    setTimeout(() => {
-      wrapper.update();
-      expect(fetch.mock.calls.length).toBe(3);
-      done();
-    }, 0)
+  test('Clicking a Person component should call fetch()', () => {
+    wrapper.find('div.headshot').first().simulate('click');
+    expect(fetch.mock.calls.length).toBe(3);
```

Comment:
just so I'm understanding this test correctly, the '3' specifies the three fields in your DB (_id, name, thumbnail_url) that will be returned when your fetch/get request is made, right?

Reply:
@HBMattoon 3 is the number of times the fetch mock is called - first for the test setup, then to request the initial movie (beforeEach), and lastly to retrieve the repertoire for the clicked personnel. In my next PR the simulated 'click' will be scoped to the test.